wai-app-static/test/WaiAppEmbeddedTest.hs

```diff
@@ -32,7 +32,7 @@ embSpec = do
                              { requestHeaders = [("If-None-Match", "Etag 1")] }
             assertStatus 304 req
 
-        it "ssIndices works" $ do
+        it "ssIndices work" $ do
```

Comment:
I thought it sounded better. But here's the reverting PR: https://github.com/yesodweb/wai/pull/574